nixos/roles/postgresql.nix

```diff
@@ -42,6 +42,20 @@ with builtins;
     })
 
     {
+      flyingcircus.roles.statshost.prometheusMetricRelabel = [
+        {
+          source_labels = [ "__name__" "datname" ];
+          regex = "postgresql_.+;(.+)-[a-f0-9]{12}";
+          replacement = "$1";
+          target_label = "datname";
```

Comment:
Ah, yes. That's a good idea. This way we fold it together to one label and do not remove the entire metric.